service/config_provider.go

```diff
@@ -182,14 +183,22 @@ func (cm *configProvider) Shutdown(ctx context.Context) error {
 	return errs
 }
 
+var fileRegexp = regexp.MustCompile("^[A-z]:")
```

Comment:
s/fileRegexp/driverLetterRegexp?

Consider to add the link to the RFC.

Reply:
Done